boards/deprecated.cmake

```diff
@@ -14,4 +14,4 @@
 
 set(nrf52840_pca20035_DEPRECATED thingy91_nrf52840)
 set(nrf9160_pca20035_DEPRECATED thingy91_nrf9160)
-set(nrf9160_pca20035ns_DEPRECATED thingy91_nrf9160ns)
+set(nrf9160_pca20035ns_DEPRECATED thingy91_nrf9160_ns)
```

Comment:
This is referencing the previous deprecated board name. This should be changed to
`set(thingy91_nrf9160ns_DEPRECATED thingy91_nrf9160_ns)`

Reply:
Thanks, sorry; that was my script.